packages/ipfs-unixfs-exporter/src/index.ts

```diff
@@ -47,9 +47,37 @@ export type ExporterProgressEvents =
   ProgressEvent<'unixfs:exporter:walk:raw', ExportWalk>
 
 export interface ExporterOptions extends ProgressOptions<ExporterProgressEvents> {
+  /**
+   * An optional offset to start reading at.
+   *
+   * If the CID resolves to a file this will be a byte offset within that file,
+   * otherwise if it's a directory it will be a directory entry offset within
+   * the directory listing. (default: undefined)
+   */
   offset?: number
+
+  /**
+   * An optional length to read.
+   *
+   * If the CID resolves to a file this will be the number of bytes read from
+   * the file, otherwise if it's a directory it will be the number of directory
+   * entries read from the directory listing. (default: undefined)
+   */
   length?: number
+
+  /**
+   * This signal can be used to abort any long-lived operations such as fetching
+   * blocks from the network. (default: undefined)
+   */
   signal?: AbortSignal
+
+  /**
+   * When a DAG layer is encountered, all child nodes are loaded in parallel but
+   * processed as they arrive. This allows us to load sibling nodes in advance
+   * of yielding their bytes. Pass a value here to control the amount of blocks
+   * loaded in parallel. (default: undefined)
+   */
+  blockReadConcurrency?: number
```

Comment:
```suggestion

  /**
   * When a DAG layer is encountered, all child nodes are loaded in parallel but
   * processed as they arrive. This allows us to load sibling nodes in advance
   * of yielding their bytes. Pass a value here to control the amount of blocks
   * loaded in parallel. I.e. the default behavior loads nodes in BFS. You can set `blockReadConcurrency: 1` to load nodes in a DFS manner. (default: undefined)
   */
  blockReadConcurrency?: number
```

Reply:
This probably goes too far, it's more like - in some circumstances this will result in a breadth-first, or breadth-first-like loading order. You can be more descriptive with the second part of this though.

Maybe:

```suggestion

  /**
   * When a DAG layer is encountered, all child nodes are loaded in parallel but
   * processed as they arrive. This allows us to load sibling nodes in advance
   * of yielding their bytes. Pass a value here to control the number of blocks
   * loaded in parallel. If a strict depth-first traversal is required, this
   * value should be set to `1`, otherwise the traversal order will tend to
   * resemble a breadth-first fan-out and yield a have stable ordering.
   * (default: undefined)
   */
  blockReadConcurrency?: number
```

I think that would be more accurate?